modules/client/src/channelProvider.ts

```diff
@@ -87,8 +87,8 @@ export class CFCoreRpcConnection extends ConnextEventEmitter implements IRpcConn
         result = await this.signWithdrawCommitment(params.message);
         break;
       case chan_nodeAuth:
```

Comment:
This is renamed to `signDigest` and consolidated with `signWithdrawalCommitment` in the NATS PR. Would be good to see how it's done there.

Reply:
well basically signWithdrawlCommitment was already signDigest anyway so it wont be a problem to converge the two